core/chaincode/handler_test.go

```diff
@@ -2550,7 +2550,7 @@ var _ = Describe("Handler", func() {
 		})
 
 		It("sends an execute message to the chaincode with the correct proposal", func() {
-			expectedMessage := *incomingMessage
+			expectedMessage := proto.Clone(incomingMessage).(*pb.ChaincodeMessage)
```

Comment:
I think the answer here is just to reference the protobuf message as a pointer rather than dereferencing it and causing a copy-by-value. I'm not sure there is any need to clone the message

Reply:
fix